.travis.yml

```diff
@@ -9,15 +9,15 @@ env:
 matrix:
   include:
     - php: 7.1
-      env: DB=MYSQL RECIPE_VERSION=4.2.x-dev PHPUNIT_TEST=1 PHPCS_TEST=1
+      env: DB=MYSQL RECIPE_VERSION="~4.2.0" PHPUNIT_TEST=1
     - php: 7.1
-      env: DB=MYSQL RECIPE_VERSION=4.3.x-dev PHPUNIT_TEST=1
+      env: DB=MYSQL RECIPE_VERSION="~4.3.0" PHPUNIT_TEST=1
```

Comment:
We should only use stable constraints when required (e.g. when the branch is gone) - otherwise we're less likely to catch breakages introduced by minor core branches until after they're released.

Reply:
agree. fixed

Reply:
Worth noting that I've been pushing and approving other builds that [do something similar](https://github.com/silverstripe/silverstripe-controllerpolicy/pull/28/files), but mainly in cases where that change is actually expanding the matrix, rather than just modifying it (effectively, my position is that we should be expanding coverage, not reducing it).